### PR TITLE
update readme with instructions

### PR DESCRIPTION
**Pull request by zramsay**
refs: #9

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,4 +2,451 @@
...
+For this step, we need two things: one or more solidity contracts, and an `epm.yaml`.
+
+Let's take a simple example, found in [this directory](monax/tests/jobs_fixtures/app06-deploy_basic_contract_and_different_solc_types_packed_unpacked/).
+
+The `epm.yaml` should look like:
+
+```
+jobs:
+
+- name: deployStorageK
```
Comment (silasdavis) [inline README.md]: Can we annotate this

**Comment by silasdavis** [comment]
@zramsay note this is still marked WIP, is there anything you wanted to add? I'd like to get it merged. There are few tweaks it needs from slightly update CLI but I can make those.

**Comment by tkuhrt** [comment]
Really interested in seeing this committed so that we can add it to an upcoming web page that links off to Getting Started tutorials for each of our projects.

**Comment by zramsay** [comment]
Updated with latest commands. Smoke tested the procedure and everything works. This script:

https://gist.github.com/zramsay/1dfe058f7bd19afe459ae9fdf0cf46c1

works out of the box. (Users just need to manually find and set the validator address on the last line)

File: README.md (hunk 1 above)
Comment (zramsay) [reply README.md]: will do in #41

**Comment by silasdavis** [comment]
Thanks @zramsay

**Pull request merged**